### PR TITLE
#2505 - Upgrade Axios Package

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Upgraded Axios to 1.6.2 version;
- Replaced FileUploadProgressEventArgs by AxiosProgressEvent;
- Tested the application modules to see if any issued with the upgrade;
- Waiting for the merge to check that the security issues related will be resolved.

### CODE REVIEW

**Review by dheepak-aot — APPROVED** [review]
Looks Good 👍

File: sources/packages/web/src/services/EducationProgramOfferingService.ts [hunk 1]
```diff
@@ -297,7 +297,7 @@ export class EducationProgramOfferingService {
   async offeringBulkInsert(
     file: Blob,
     validationOnly: boolean,
-    onUploadProgress: (progressEvent: FileUploadProgressEventArgs) => void,
+    onUploadProgress: (progressEvent: AxiosProgressEvent) => void,
```
Comment (ann-aot) [inline sources/packages/web/src/services/EducationProgramOfferingService.ts:300]: just wanted to double-check, I see that, `bytes` and `loaded` are the 2 mandatory properties in AxiosProgressEvent https://www.jsdocs.io/package/axios#AxiosProgressEvent, and  `bytes` was not in FileUploadProgressEventArgs. So, I belive, with this new package update `bytes` will also be part of the `progressEvent` right?

**Review by ann-aot — APPROVED** [review]
LGTM @andrepestana-aot 👍  Nice work, just a qn

File: testing/package-lock.json
Comment (andrewsignori-aot) [inline testing/package-lock.json:1]: I believe that this file was created by mistake. Can you please double-check?

Comment (andrepestana-aot) [reply testing/package-lock.json:1]: Yes, I removed that one. Thanks!

File: sources/packages/web/src/services/EducationProgramOfferingService.ts (hunk 1 above)
Comment (andrepestana-aot) [reply sources/packages/web/src/services/EducationProgramOfferingService.ts:300]: Yes, you're right. https://github.com/axios/axios?tab=readme-ov-file#-progress-capturing

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrepestana-aot

**Review by andrewsignori-aot — APPROVED** [review]
Nice work, looks good 👍

**Pull request merged**